circus-train-avro/src/test/java/com/hotels/bdp/circustrain/avro/util/FileSystemPathResolverTest.java

```diff
@@ -73,7 +81,7 @@ public void resolveNameServicesAddsAuthorityToPathWithoutScheme() {
     setDfsPaths("foo");
     Path input = new Path("/etl/test/avsc/schema.avsc");
     Path result = resolver.resolveNameServices(input);
-    Path expected = new Path("/foo/etl/test/avsc/schema.avsc");
+    Path expected = new Path("/etl/test/avsc/schema.avsc");
```

Comment:
@abhimanyugupta07  does this seem correct to you?

Reply:
I guess previously if there was no schema it would assume that HDFS was in use whereas s3 paths always start with s3. I think this is usually the case with our on-prem setup so I'm not sure if the change you made to always check `hdfs` will work for those cases...

Reply:
it does this test tests the two methods separately but in our code we actually always add the schema and then add the name. So we won't call it with "/" it will be hdfs:// or file:// depending on the HiveConf.
I think the code as it is now where we only add the name node if and only if we're certain that we're on hdfs makes more sense.